app/components/Settings/Preview/ChartEditor.react.js

```diff
@@ -58,6 +66,8 @@ export default class ChartEditor extends PureComponent {
                 <div style={controlPanelStyle}>
                     <div style={dropdownContainerStyle}>
                         <div style={columnSelectLabelStyle}>{columnLabel}</div>
+                        {/* TODO is this a bug?
+                        there's no value attribute on <select> (instead use option's selected attr) */}
```

Comment:
Yeah you're right, I'm not sure what's going on here. I think @jackparmer wrote these lines